examples/prism-cmake.html

```diff
@@ -0,0 +1,86 @@
+<h2>Comments</h2>
+<pre><code># This is a comment</code></pre>
+
+<h2>Functions</h2>
+<pre><code>add_library(foo main.cpp)
+target_link_libraries(foo bar)
+</code></pre>
+
+<h2>Keywords</h2>
+<pre><code>CMAKE_COMPILER_IS_GNUG77
+CMAKE_COMPILE_PDB_OUTPUT_DIRECTORY
+CMAKE_COMPILE_PDB_OUTPUT_DIRECTORY_\w+
+CMAKE_CONFIGURATION_TYPES
+CMAKE_CPACK_COMMAND
+CMAKE_CROSSCOMPILING
+CMAKE_CROSSCOMPILING_EMULATOR
+CMAKE_CTEST_COMMAND
+CMAKE_CUDA_EXTENSIONS
+CMAKE_CUDA_HOST_COMPILER
+CMAKE_CUDA_SEPARABLE_COMPILATION
+CMAKE_CUDA_STANDARD
+CMAKE_CUDA_STANDARD_REQUIRED
+</code></pre>
+
+<h2>Properties</h2>
+<pre><code>CUDA_STANDARD
+CUDA_STANDARD_REQUIRED
+CXX_EXTENSIONS
+CXX_STANDARD</code></pre>
+
+<h2>Full example</h2>
+<pre><code>cmake_minimum_required(VERSION 3.13)
+
+        project(crypto)
```

Comment:
Do we really want this indentation here?

Reply:
Of course not :)

Reply:
I somehow couldn't get it to work. How to see if this example works? When I open examples.html, it says that CMake has no examples.

Reply:
No, unfortunately, we cant.
The examples page gets the info of what examples are available from Github using the master branch of this project. So it won't be accessible until it gets merged.

I know, not the best setup but you can always the [test page](https://github.com/PrismJS/prism/blob/master/test.html) of your local project to see the results.

Reply:
Ok, sorry about missing this answer earlier.